src/Resources/config/controller.yaml

```diff
@@ -1,4 +1,17 @@
 services:
     SilasJoisten\Sonata\MultiUploadBundle\Controller\MultiUploadController:
-        arguments: ['@sonata.media.manager.media']
+        arguments:
+            $mediaManager: '@sonata.media.manager.media'
```

Comment:
You should add a recipe for this bundle 👍🏻

Reply:
There is already a receipe